Assets/HoloToolkit/Input/Scripts/Utilities/Interactions/TapToPlace.cs

```diff
@@ -216,31 +202,30 @@ private static Vector3 GetPlacementPosition(Vector3 headPosition, Vector3 gazeDi
         /// </summary>
         /// <param name="origin">Origin of the raycast</param>
         /// <param name="direction">Direction of the raycast</param>
-        /// <param name="spatialMapHit">Result of the raycast when a hit occurred</param>
-        /// <returns>Whether it found a hit or not</returns>
-        private static bool SpatialMappingRaycast(Vector3 origin, Vector3 direction, out RaycastHit spatialMapHit)
+        /// <param name="spatialMapHit">Result of the raycast when a hit occured</param>
+        /// <returns>Wheter it found a hit or not</returns>
+        private bool UpdateRaycast(Vector3 origin, Vector3 direction, out RaycastHit hitInfo)
         {
-            if (SpatialMappingManager.Instance != null)
+            if (GazeManager.Instance != null)
```

Comment:
Should we be using the Focus Manager here?

How does the new pointer system fit into this?

Reply:
I left this for the separate PR as you suggested above for adding features. Just moved the script to SpatialMapping since it uses it.